cw_core/lib/get_height_by_date.dart

```diff
@@ -300,6 +305,21 @@ DateTime getDateByBitcoinHeight(int height) {
   return estimatedDate;
 }
 
+int getLitecoinHeightByDate({required DateTime date}) {
+  String dateKey = '${date.year}-${date.month.toString().padLeft(2, '0')}';
+  final closestKey = litecoinDates.keys
+      .firstWhere((key) => formatMapKey(key).isBefore(date), orElse: () => litecoinDates.keys.last);
+  final beginningBlock = litecoinDates[dateKey] ?? litecoinDates[closestKey]!;
+
+  final startOfMonth = DateTime(date.year, date.month);
+  final daysDifference = date.difference(startOfMonth).inDays;
+
+  // approximately 6 blocks per hour, 24 hours per day
+  int estimatedBlocksSinceStartOfMonth = (daysDifference * 24 * 6);
```

Comment:
this I believe only applies to Bitcoin, so let's disable rescan from date until we get the correct way or dates of Litecoin